php/src/vaas/Vaas.php

```diff
@@ -272,14 +242,8 @@ public function forUrlAsync(string $uri, ?ForUrlOptions $options = null, ?Cancel
             $this->logger->debug("Requesting verdict for URL: $uri");
             $uri = Vaas::validUri($uri);
 
-            if ($options === null) {
-                $options = new ForUrlOptions(
-                    [
-                        'vaasRequestId' => null,
-                        'useHashLookup' => $this->options->useHashLookup ?? true,
-                    ]
-                );
-            }
+            $options = $options ?? ForUrlOptions::default();
```

Comment:
Same here as before:
```suggestion
            if ($options === null) {
                $options = new ForUrlOptions($this->options->useHashLookup, null);
            }
```